code/game/objects/items/storage/storage.dm

```diff
@@ -171,6 +159,12 @@
 /obj/item/storage/proc/open(mob/user)
 	if(user.s_active == src) //Spam prevention.
 		return
+
+	if(istype(loc, /obj/item/storage) && required_skill_for_nest_opening)
+		if(!user || !user.skills || user.skills.get_skill_level(required_skill_for_nest_opening) < required_skill_level_for_nest_opening)
```

Comment:
```suggestion
		if(!user || user.skills?.get_skill_level(required_skill_for_nest_opening) < required_skill_level_for_nest_opening)
```
Why not do it this way? The only time somebody should have null skills is if given it by an admin, and that is what is used to give someone all skills.

Reply:
Errr wrong spot but sure